app/models/group.js

```diff
@@ -14,7 +14,7 @@ export default Model.extend({
     tags: attr('string'),
     dateCreated: attr('date'),
     dateUpdated: attr('date'),
-    collection: belongsTo('collection-base'),
```

Comment:
think this was old

Reply:
should this be deleted or simply changed to 'collection'?

Reply:
good catch!